sparse_strips/vello_common/src/encode.rs

```diff
@@ -375,15 +403,15 @@ fn encode_stops(stops: &[ColorStop], start: f32, end: f32, pad: bool) -> Vec<Gra
             factors[i] = c1_minus_c0 / x1_minus_x0;
         }
 
-        GradientRange {
+        vec![GradientRange {
             x0,
             x1,
             c0,
             factors,
-        }
+        }]
```

Comment:
This can return `GradientRange` directly rather than allocating a `Vec`:

```suggestion
        GradientRange {
            x0,
            x1,
            c0,
            factors,
        }
```

Reply:
(Cont'd in https://github.com/linebender/vello/pull/931#discussion_r2056146844.)